From 8623ba181ad7bfcf6b82b8353ef2b20328b9076d Mon Sep 17 00:00:00 2001
From: Michael Fladischer <fladi@debian.org>
Date: Thu, 8 Oct 2015 08:50:25 -0700
Subject: Remove pretty-printing nose-cover3 from build.

 nose-cover3 is not essential to the build-proccess and is not packaged
 for Debian. Thus removing it from the sources.
Forwarded: not-needed
Last-Update: 2011-11-28
Patch-Name: fix_dependencies.patch
---
 requirements/test.txt | 1 -
 setup.py              | 7 -------
 tests/settings.py     | 3 ---
 3 files changed, 11 deletions(-)

diff --git a/requirements/test.txt b/requirements/test.txt
index 9e63952..f2e10a0 100644
--- a/requirements/test.txt
+++ b/requirements/test.txt
@@ -1,7 +1,6 @@
 unittest2>=0.4.0
 coverage>=3.0
 nose
-nose-cover3
 mock
 django-nose
 python-memcached
diff --git a/setup.py b/setup.py
index 31ca26c..ce02fb7 100755
--- a/setup.py
+++ b/setup.py
@@ -147,16 +147,9 @@ class CIRunTests(RunTests):
     def extra_args(self):
         toxinidir = os.environ.get('TOXINIDIR', '')
         return [
-            '--with-coverage3',
-            '--cover3-xml',
-            '--cover3-xml-file=%s' % (
-                os.path.join(toxinidir, 'coverage.xml'), ),
             '--with-xunit',
             '--xunit-file=%s' % (
                 os.path.join(toxinidir, 'nosetests.xml'), ),
-            '--cover3-html',
-            '--cover3-html-dir=%s' % (
-                os.path.join(toxinidir, 'cover'), ),
         ]
 
 
diff --git a/tests/settings.py b/tests/settings.py
index 73b5d2e..9a52cb7 100644
--- a/tests/settings.py
+++ b/tests/settings.py
@@ -43,9 +43,6 @@ COVERAGE_EXCLUDE_MODULES = (
 NOSE_ARGS = [
     os.path.join(here, os.pardir, 'djcelery', 'tests'),
     os.environ.get('NOSE_VERBOSE') and '--verbose' or '',
-    '--cover3-package=djcelery',
-    '--cover3-branch',
-    '--cover3-exclude=%s' % ','.join(COVERAGE_EXCLUDE_MODULES),
 ]
 
 BROKER_URL = 'amqp://'
